apps/cowswap-frontend/src/modules/volumeFee/state/cowswapFeeAtom.ts

```diff
@@ -45,8 +45,8 @@ export const cowSwapFeeAtom = atom((get) => {
   if (chainId === SupportedChainId.ARBITRUM_ONE && !shouldApplyFee(account, featureFlags.arb1CowSwapFeePercentage))
     return null
 
-  const isInputTokenStable = STABLECOINS[chainId].includes(getCurrencyAddress(inputCurrency).toLowerCase())
-  const isOutputTokenStable = STABLECOINS[chainId].includes(getCurrencyAddress(outputCurrency).toLowerCase())
+  const isInputTokenStable = STABLECOINS[chainId].has(getCurrencyAddress(inputCurrency).toLowerCase())
+  const isOutputTokenStable = STABLECOINS[chainId].has(getCurrencyAddress(outputCurrency).toLowerCase())
```

Comment:
🤦 

Thank you!

Reply:
I had it locally, but forgot to commit it 
![image](https://github.com/user-attachments/assets/442096cc-30d7-469a-bae5-01b4df5f2926)